compiler_setup/xilinx.sh

```diff
@@ -1,16 +1,18 @@
 # Environment variables for OpenCL for Xilinx FPGAs
 
-export PLATFORM = xilinx_u200_xdma_201830_2
-export EXECUTION_TARGET = sw_emu # Options: sw_emu | hw_emu | hw
-export OCL_COMPILER = "v++ --compile --platform ${PLATFORM} --target ${EXECUTION_TARGET}"
-export OCL_LINKER = "v++ --link --platform ${PLATFORM} --target ${EXECUTION_TARGET}"
+export PLATFORM=xilinx_u200_xdma_201830_2
+export EXECUTION_TARGET=hw # Options: sw_emu | hw_emu | hw
```

Comment:
Should that be `?=` to allow the user to override the setting, as suggested by the messages `echo`d below?

Reply:
`?=` is a Makefile operator, not bash :)

The target can be changed after the "source" command to the preferred option before executing the make.

Reply:
Oh yes, I knew that :-)